### PR TITLE
[FrameworkBundle][TwigBundle] slightly clarify command descriptions

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 class YamlLintCommand extends BaseLintCommand
 {
     protected static $defaultName = 'lint:yaml';
-    protected static $defaultDescription = 'Lints a file and outputs encountered errors';
+    protected static $defaultDescription = 'Lints a YAML file and outputs encountered errors';
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php:27]: I think the base lint commandalso needs the same patch, isn't it? (same for xliff)

Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php:27]: good catch, I changed them too

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php:27]: same for the twig one :)

Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php:27]: already done in same push ;)

**Review by nicolas-grekas — APPROVED** [review]

**Pull request merged**